content/en/guides/hosting/hosting-options/self-managed/kubernetes-operator/operator-airgapped.md

```diff
@@ -80,10 +80,10 @@ Operator Images:
   wandb/controller:1.16.1
 W&B Images:
   wandb/local:0.62.2
-  docker.io/bitnami/redis:7.2.4-debian-12-r9
-  quay.io/prometheus-operator/prometheus-config-reloader:v0.67.0
-  quay.io/prometheus/prometheus:v2.47.0
-  otel/opentelemetry-collector-contrib:0.97.0
+  redis:7.2
+  quay.io/prometheus-operator/prometheus-config-reloader:v0.67
+  quay.io/prometheus/prometheus:v2.47
+  otel/opentelemetry-collector-contrib:0.97
```

Comment:
The same goes for these images. Our Helm charts are pre-configured with the Bitnami Helm charts, hence their images.
Replacing the images requires a test. Better to sync with the Platform team before releasing this change publicly.